anycloud/cli/src/main.rs

```diff
@@ -17,6 +17,9 @@ pub async fn main() {
     .subcommand(SubCommand::with_name("new")
       .about("Deploys your repository to a new App with a Deploy Config from anycloud.json")
       .arg_from_usage("-e, --env-file=[ENV_FILE] 'Specifies an optional environment file'")
+      .arg_from_usage("[NON_INTERACTIVE] -n, --non-interactive 'Specifies an optional flag for non interactive CLI mode'")
```

Comment:
This reads a bit weird "specifies an optional flag".

Maybe:

```suggestion
      .arg_from_usage("[NON_INTERACTIVE] -n, --non-interactive 'Enables non interactive CLI mode useful for scripting'")
```